spec/integration/rails/mailer_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+RSpec.describe Loga::LogSubscribers::ActionMailer do
+  context 'when an email is being sent' do
+    it 'delivers an email' do
+      send_mail = -> { FakeMailer.send_email }
+
+      expect(&send_mail).to change { FakeMailer.deliveries.size }.by(1)
+    end
+
+    it 'creates an "info" log' do
+      expect(Loga.logger).to receive(:info)
```

Comment:
💁 Could this test the actual payload? There example integration specs [here](https://github.com/FundingCircle/loga/blob/2e48c24164977b7720e26d4617f26507d4197ed2/spec/integration/rails/request_spec.rb)